src/main/java/com/cburch/logisim/fpga/data/IOComponentTypes.java

```diff
@@ -66,6 +66,10 @@
    * Open and constant are used in the map dialog to be able to map IOcomponents
    * to a constant or a open (hence no connection external of the FPGA/CPLD).
    */
+  
+  public static final int rotationZero = 0;
```

Comment:
Can we have `Ninety` in var names be replaced with `90` and `Zero` with `0`? That's faster to read. If so, the same for translation string keys (`dipSwitchMinusNinety` etc).

Reply:
Yes and no, -90 cannot. Hence it would be inconsistent. see also my remark below